modules/floating-buttons/module.php

```diff
@@ -519,13 +541,14 @@ private function render_floating_buttons(): void {
 
 		foreach ( $query->posts as $post_id ) {
 			$conditions = get_post_meta( $post_id, '_elementor_conditions', true );
+
 			if ( ! $conditions ) {
 				continue;
 			}
+
 			if ( in_array( 'include/general', $conditions ) ) {
 				$document = Plugin::$instance->documents->get( $post_id );
 				$document->print_content();
-				break;
```

Comment:
Was removing this `break` intentional?

Reply:
@louiswol94 yes because now we have two possible elements, one floating button and one floating bar, which are set as entire site

Reply:
And this will not cause more than one of each to show up?